service/enry/scanner_test.go

```diff
@@ -0,0 +1,176 @@
+package enry
+
+import (
+	"testing"
+
+	"github.com/src-d/lookout"
+	"github.com/src-d/lookout/mock"
+	"github.com/stretchr/testify/suite"
+)
+
+type ScannerSuite struct {
+	suite.Suite
+}
+
+func TestScannerSuiteSuite(t *testing.T) {
+	suite.Run(t, new(ScannerSuite))
+}
+
+func (s *ScannerSuite) TestFileChangeVendorScanner() {
+	require := s.Require()
+
+	underlyingChanges := []*lookout.Change{
+		// non-vendor files should survive
+		{
+			Head: &lookout.File{Path: "f1new.go"},
+		},
+		{
+			Base: &lookout.File{Path: "f1old.py"},
+		},
+		{
+			Base: &lookout.File{Path: "f2old.py"},
+			Head: &lookout.File{Path: "f2new.js"},
+		},
+		// the change that used to be vendor but isn't anymore should survive
```

Comment:
Is this comment correct? The test case seems to be the exact opposite

Reply:
thanks! The comment is correct but the code is wrong, the right one was at the very end of use cases. Fixed.